src/Swashbuckle.AspNetCore.Filters/Swashbuckle.AspNetCore.Filters.csproj

```diff
@@ -27,6 +27,7 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Extensions.DependencyInjection" Version="2.0.0" />
+    <PackageReference Include="Microsoft.OpenApi" Version="1.1.3" />
```

Comment:
Just wondering why you've added a PackageReference to this, I guess it is added by Swashbuckle already?

Reply:
Yes, it was added by Swashbuckle already but that version isn't compatible with the `OpenApiRawString` class. There've got a strange way of handling `OpenApiPrimatives` that causes problems when casting using implicit operators. They've added better handling of polymorphism in the newer version which I'm hoping will be added to Swashbuckle in the future and we can remove this reference.